source/website/src/views/Step3.vue

```diff
@@ -292,6 +299,79 @@ SPDX-License-Identifier: Apache-2.0
         this.column_type_options.forEach(x => x.disabled = false)
         this.pii_type_options.forEach(x => x.disabled = false)
       },
+      onExport() {
+        if (!this.validateForm()) return
+        const a = document.createElement("a");
+        const file = new Blob([JSON.stringify({"columns": this.items })], { type:  "application/json"});
+        a.href = URL.createObjectURL(file);
+        a.download = "amcufa_exported_schema_" + Date.now() + ".json";
+        a.click();
+        console.log("Schema Exported: ".concat(JSON.stringify({"columns": this.items })))
+      },
+      onBrowseImports(){
+        if (confirm("WARNING: This will reset the columns!")) {
+          document.getElementById('importFile').click()
+        }
+      },
+      onImport(e){
+          let files = e.target.files || e.dataTransfer.files;
+          if (!files.length) return;
+          let reader = new FileReader();
+          reader.onload = e => {
+            console.log("Imported schema: ".concat(e.target.result))
+            let importJson = JSON.parse(e.target.result);
+            this.$refs['file'].reset()
+            if(importJson.constructor != Object){
+              alert("Invalid Schema: Expecting a dictionary.")
```

Comment:
Would it be better to use a `<b-alert>` instead when throwing these errors for UI design consistency since that is how we throw all our other errors in `Step3`?

Reply:
researching on <b-alert>

Reply:
I have applied PR reviews https://github.com/aws-solutions/amazon-marketing-cloud-uploader-from-aws/pull/69/commits/f18b43ae634a62805b6e050a1e5c74881fd0fd3a

Reply:
![image](https://user-images.githubusercontent.com/7339334/215626183-c0c68d3f-e17e-419b-983a-f32bd750f2b6.png)